setup.py

```diff
@@ -1,48 +0,0 @@
-from pathlib import Path
-
-from setuptools import find_packages
-from setuptools import setup
-
-about = {}
-with open(Path(__file__).parent / "bro" / "__version__.py", "r") as f:
-    exec(f.read(), about)
-
-long_description = (Path(__file__).parent / "README.md").read_text()
-long_description = long_description.replace("X.Y.Z", about["__version__"])
```

Comment:
this not covered by the new pyproject.toml now. I would suggest to change it into a dynamic badge: https://img.shields.io/github/v/tag/viktor-platform/bro?sort=date

@regbers-viktor: not sure if there was another reason to do this static?